examples/autoconfigure/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java'
+}
+
+description = "OpenTelemetry Examples for SDK autoconfiguration"
+ext.moduleName = "io.opentelemetry.examples.autoconfigure"
+
+dependencies {
+    implementation("io.opentelemetry:opentelemetry-api")
+    implementation("io.opentelemetry:opentelemetry-exporter-logging")
+    implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
+
+    //alpha modules
+    implementation("io.opentelemetry:opentelemetry-api-metrics")
```

Comment:
This isn't needed now that metrics is out, correct/

Reply:
Shouldn't be after it points to snapshot :)

Reply:
Good catch, removed.